ddtrace/tracer/tracer_test.go

```diff
@@ -571,6 +571,44 @@ func TestTracerSpanGlobalTags(t *testing.T) {
 	assert.Equal("value", child.Meta["key"])
 }
 
+func TestTracerSpanServiceMappings(t *testing.T) {
+	assert := assert.New(t)
+
+	t.Run("Service from StartOption", func(t *testing.T) {
```

Comment:
In Go, the test name is not supposed to be a description of the test, as in other languages (like for example Ruby, Node, Java, and many others). It's meant to be just an identifier. One word. In some rare occasions, perhaps two words separated by a dash, slash, or underscore, but not spaces. This is made even more obvious when you run the tests using `-v` and notice that all the spaces are replaced with underscores and it looks plain weird.

The name of the test is supposed to be simply a _hint_ of how to find the failing test in your suite. See also: https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#tests

Some good test names in your case are (for all of your 5 tests):
* `WithServiceMapping`
* `child`
* `StartSpanOption`
* `tag`
* `globalTags`

I hope it makes sense.

Reply:
Ah, I was not aware of this. Thank you, this definitely makes sense! Noted for future PR's.

Reply:
Made this change 👍